acq4/devices/MIESPatchPipette/testpulse.py

```diff
@@ -20,8 +20,9 @@ def __init__(self, dev, params):
 
     def newTestPulse(self, data):
         """Got the signal from MIES that data is available, update"""
-        tp = TestPulse(self.dev, {}, data[:, self._headstage])
+        tp = TestPulse(self.dev, data[:, self._headstage])
```

Comment:
The neuroanalysis package has a `MiesTestPulse` class. I don't know if that's something that would be useful in this case, necessarily, but if test pulse handling starts giving you problems, it might be worth investigating. (Or maybe you already know about it, but I figured it wouldn't hurt to mention it)

Reply:
MiesTestPulse is meant for reading from MIES-generated NWB files; not useful here